### PR TITLE
Docs for KUDO-kubeflow 1.0.1-0.4.0

### PR DESCRIPTION
**Pull request by porridge**
## Jira Ticket
https://jira.d2iq.com/browse/D2IQ-70590

## Description of changes being made

Updating docs for new release.

Estimated merge day: 2020-08-05

## Checklist
- [x] Change all affected versions, if applicable (e.g. 1.13, 2.0, 2.1).
- [x] Test all commands and procedures, if applicable.
- [x] Create your PR against `staging`, not `master`. 
- [x] Provide an estimated date for deploying the doc change. Note: Improvements or fixes can be merged ASAP. 
- [x] Add [redirects](https://github.com/mesosphere/dcos-docs-site/wiki/Redirects) if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> ie `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/wiki/Contributing) for more information.

### CODE REVIEW

**Comment by cmastr** [comment]
Looks very nice! 
0332E63D-C24D-4A86-BDAB-0245B18BD33E_1_105_c

File: pages/ksphere/kubeflow/1.0.1-0.4.0/download/index.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+layout: layout.pug
+navigationTitle: Download
+title: Download KUDO for Kubeflow
+menuWeight: 1
```
Comment (cneth) [inline pages/ksphere/kubeflow/1.0.1-0.4.0/download/index.md:5]: Is this really where we want this page to appear in the side bar? (The menuweight determines the order, the lower the value, the 'higher' in the side bar it appears)

As it stands, this commit results in the following list in the left hand side bar:

"Download"
"Introducing KUDO for Kubeflow"
"Legal Notices"
"Manage Secrets"
"Manage Users"
"Tutorials"

I would think we would want "Introducing KUDO for Kubeflow" to be first and then perhaps 'Download' second.   I would think "Legal Notices" would be last.   

Also, it looks like only the first 4 items in the list appear on the main 'ksphere' page, so presumably we don't want to have 'legal notices' as one of the 4 links on the main ksphere page...

File: pages/ksphere/kubeflow/1.0.1-0.4.0/index.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+---
+layout: layout.pug
+navigationTitle: KUDO for Kubeflow Technical Preview
+title: Welcome to KUDO for Kubeflow Technical Preview
+beta: false
+menuWeight: 1
+---
+
+![KUDO for Kubeflow](img/d2kf-overview.png)
```
Comment (cneth) [inline pages/ksphere/kubeflow/1.0.1-0.4.0/index.md:9]: I built the docs site, but I don't see this page anywhere in the resulting site.  Where is it supposed to be?

File: pages/ksphere/kubeflow/1.0.1-0.4.0/download/index.md (hunk 1 above)
Comment (porridge) [reply pages/ksphere/kubeflow/1.0.1-0.4.0/download/index.md]: Good catch! Adjusted, PTAL.

File: pages/ksphere/kubeflow/1.0.1-0.4.0/index.md (hunk 2 above)
Comment (porridge) [reply pages/ksphere/kubeflow/1.0.1-0.4.0/index.md:9]: Hm the page is there, as it [is for the previous version](https://docs.d2iq.com/ksphere/kubeflow/1.0.1-0.3.1/). It just does not seem to be linked to from anywhere? Perhaps it only gets linked to if there is more than one version of kubeflow present? @cmastr do you know?
It's a pity that the nobody gets to see the diagram in that case. cc @akirillov

File: pages/ksphere/kubeflow/1.0.1-0.4.0/download/index.md (hunk 1 above)
Comment (cneth) [reply pages/ksphere/kubeflow/1.0.1-0.4.0/download/index.md]: Looks much better!

File: pages/ksphere/kubeflow/1.0.1-0.4.0/index.md (hunk 2 above)
Comment (cmastr) [reply pages/ksphere/kubeflow/1.0.1-0.4.0/index.md:9]: Looking at this. Will see if I can add it to the TOC.

Comment (porridge) [reply pages/ksphere/kubeflow/1.0.1-0.4.0/index.md:9]: Any luck with this? If not, then perhaps we can postpone this to later? We had a similar unreachable page in the previous release as well...

**Comment by porridge** [comment]
From kubeflow team perspective we can merge these docs now @cmastr

**Comment by porridge** [comment]
One more thing: I don't know if it applies here, but I could not find this page mentioned in the checklist:
>  Add release date to Release Notes page in the following format: was released on , ie Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020

File: pages/ksphere/kubeflow/1.0.1-0.4.0/index.md (hunk 2 above)
Comment (cmastr) [reply pages/ksphere/kubeflow/1.0.1-0.4.0/index.md:9]: Yeah I didn't get very far with this so we'll have to hold off for now. There would have to be some restructuring of the content in the Intro page.

**Comment by cmastr** [comment]
> One more thing: I don't know if it applies here, but I could not find this page mentioned in the checklist:
> 
> > Add release date to Release Notes page in the following format: was released on , ie Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020

Doesn't apply since we are not GA yet. We don't do release notes until a 1.x product.

**Pull request merged**